static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
```

Comment:
I have a strong opinion too about which and how the guardian should choose open source libraries. As it seems Promises are the way forward (temporarily I hope), so let's embrace Promises and use them the way they're supposed to: all new async libraries should be Promise-based, and all existing ones should be "promisified". This way, the coding style will be harmonious and we can avoid anti-patterns like the ones we can see at the moment. That might mean indeed more of these proxies will be written eventually.

In any case `lazydom` seems like a good choice, as a matter of fact I raised [another PR](https://github.com/guardian/frontend/pull/11533) to do just that and will rename it that way.